rest/slurmrestapi.py

```diff
@@ -118,20 +119,38 @@ def login():
 @cache()
 def get_jobs():
     jobs = pyslurm.job().get()
+    result = {}
+    if auth_enabled:
+        """
```

Comment:
PEP-8 says that block comment should use #[1], not docstrings, please fix it.

[1] https://www.python.org/dev/peps/pep-0008/#block-comments